Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/border.h

```diff
@@ -198,8 +198,8 @@ namespace Polygon_mesh_processing {
     }
 
     //face index map given as a named parameter, or as an internal property map
-    FIMap fim = choose_param(get_param(np, CGAL::face_index),
-                             get_const_property_map(CGAL::face_index, pmesh));
+    FIMap fim = boost::choose_param(get_param(np, CGAL::face_index),
```

Comment:
1- yes but we should not have used it in the first place and have our own functions (it is not related to this bug-fix)
2- no element from the boost namespace is a parameter, ADL cannot work
3- choose_param is overloaded in the CGAL namespace IIRC